packages/apps/tools/src/components/Common/DrawerToolbar/styles.css.ts

```diff
@@ -1,66 +1,74 @@
-import { darkThemeClass, sprinkles, vars } from '@kadena/react-ui/theme';
+import { atoms, darkThemeClass, tokens } from '@kadena/react-ui/theme';
 import { style } from '@vanilla-extract/css';
 
 export const gridItemCollapsedSidebarStyle = style([
-  sprinkles({
+  atoms({
     position: 'fixed',
-    right: '$12',
-    top: '$16',
     bottom: 0,
-    backgroundColor: '$neutral1',
-    width: '$12',
     display: 'flex',
     flexDirection: 'column',
-    fontSize: '$sm',
+    fontSize: 'sm',
     zIndex: 1,
   }),
   {
+    width: tokens.kda.foundation.size.n12,
+    right: tokens.kda.foundation.size.n12,
+    top: tokens.kda.foundation.size.n16,
     height: '100vh',
-    borderLeft: `solid 1px ${vars.colors.$borderSubtle}`,
+    backgroundColor: tokens.kda.foundation.color.neutral.n1,
+    borderLeft: `solid 1px ${tokens.kda.foundation.color.border.base}`,
     transition: 'width 0.1s ease',
     selectors: {
       '&.isOpen': {
-        width: `calc(${vars.sizes.$64} + ${vars.sizes.$12})`,
+        width: `calc(${tokens.kda.foundation.size.n64} + ${tokens.kda.foundation.size.n12})`,
       },
     },
   },
 ]);
 
 export const gridItemMiniMenuStyle = style([
-  {
+  atoms({
+    height: '100%',
     flexDirection: 'column',
     position: 'relative',
-  },
+  }),
   {
-    height: '100%',
-    borderLeft: `solid 1px ${vars.colors.$borderSubtle}`,
+    borderLeft: `1px solid #E3E1E5`,
```

Comment:
I don't think we should ever use hardcoded color values. Can this be replaced with one of the tokens?